src/main.rs

```diff
@@ -227,13 +228,14 @@ fn main() {
                 }
             }))
         .after_help("Optimization levels:
-    -o 0  =>  --zc 3 --nz                  (0 or 1 trials)
-    -o 1  =>  --zc 9                       (1 trial, determined heuristically)
-    -o 2  =>  --zc 9 --zs 0-3 -f 0,5       (8 trials)
-    -o 3  =>  --zc 9 --zs 0-3 -f 0-5       (24 trials)
-    -o 4  =>  --zc 9 --zs 0-3 -f 0-5 -a    (24 trials + 6 alpha trials)
-    -o 5  =>  --zc 3-9 --zs 0-3 -f 0-5 -a  (96 trials + 6 alpha trials)
-    -o 6  =>  --zc 1-9 --zs 0-3 -f 0-5 -a  (180 trials + 6 alpha trials)
+    -o 0   =>  --zc 3 --nz                  (0 or 1 trials)
+    -o 1   =>  --zc 9                       (1 trial, determined heuristically)
+    -o 2   =>  --zc 9 --zs 0-3 -f 0,5       (8 trials with zlib or 2 trials with other compressors)
+    -o 3   =>  --zc 9 --zs 0-3 -f 0-5       (24 trials with zlib or 6 trials with other compressors)
+    -o 4   =>                               (deprecated; same as `-o 3`)
+    -o 5   =>  --zc 3-9 --zs 0-3 -f 0-5     (96 trials with zlib; same as `-o 3` for other compressors)
```

Comment:
@shssoichiro Actually, wait, I'm not sure that 96 is / was correct here: `(9 - 3 + 1) * (3 - 0 + 1) * (5 - 0 + 1) = 168`.
And the one below: `(9 - 1 + 1) * (3 - 0 + 1) * (5 - 0 + 1) = 216`.

Reply:
I think you're right. Maybe this was right at some point in the past and then the settings got changed? Or my math was just wrong to begin with. Since you're in this code already, would you mind updating that? Thanks!

Reply:
Done.